### PR TITLE
Added Screen Reader application question to Accessibility-quiz.ts

### PR DESCRIPTION
**Pull request by devesh-mishra13**
## Summary of changes

question about licensed software of screen reader for windows.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -832,6 +832,15 @@ const accessibilityQuiz = [
     Explanation:
       "WCAG guidelines are globally recognized and help create web content that is accessible and usable for individuals with disabilities, enhancing inclusivity on the web.",
     Link: "https://www.w3.org/WAI/standards-guidelines/wcag/"
+  },
+  {
+    Question: "Which of the following screen reader is available as licensed software for windows?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:837]:
```suggestion
    Question: "Which of the following screen readers is available as licensed software for Windows?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by devesh-mishra13 — COMMENTED** [review]
JAWS question added.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**